static/styles/zulip.css

```diff
@@ -685,6 +685,43 @@ li.actual-dropdown-menu i {
     width: 25px;
 }
 
+.settings-profile {
```

Comment:
This is a pretty confusing class name for something that's not in the settings UI.

Reply:
Sure, will change this into something better.